docs/index.js

```diff
@@ -245,6 +253,7 @@ const App = () => {
           id="main-content"
           workspaceRef={el => setWorkspace}
           tabIndex={-1}
+          className="Layout-with-footer"
```

Comment:
Since this a modifier of `Layout`, I suggest this:
```suggestion
          className="Layout--with-footer"
```

Reply:
Unused class I left in - removing it